docs/tutorials/screen1d.rst

```diff
@@ -0,0 +1,372 @@
+
+    scr1 = Screen1D(normal=scr1_normal, p=scr1_pos, v=scr1_vel,
+                    magnification=scr1_magnification)
+
+    print(scr1)
+
+
+The telescope
+-------------
+
+Finally, create the telescope using the :py:class:`~screens.screen.Telescope`
+class. The default argument values set its position and velocity to zero. Note
+that this object also has a ``magnification`` attribute (with a default value
+of unity), which can be thought of as the efficiency of the telescope.
+
+.. jupyter-execute::
+
+    telescope = Telescope()
+
+    print(telescope)
+
+
+Generating observations using :py:meth:`~screens.screen.Screen.observe`
+=======================================================================
+
+The :py:meth:`~screens.screen.Screen.observe` method can be used to quickly
+generate scintillometric observations. It is available on the
+:py:class:`~screens.screen.Screen` class (of which
+:py:class:`~screens.screen.Telescope` and :py:class:`~screens.screen.Screen1D`
+are subclasses) and it requires two arguments:
+
+- The ``source`` argument is the source of radiation that is being observed.
+  This should be either a :py:class:`~screens.screen.Source` object (for
+  simulating a direct observation) or a :py:class:`~screens.screen.Screen`
+  object (for simulating an observation of a screen that is scattering
+  radiation from a source behind it).
+- The ``distance`` argument is the physical distance at which ``source`` is
+  being observed. It should be an Astropy
+  :py:class:`~astropy.units.quantity.Quantity` object.
+
+For example, here we simulate a direct observation of the pulsar from the
+telescope (i.e., ignoring the screen for now). As we can see, this returns
+another :py:class:`~screens.screen.Telescope` object, but one that has a
+``source`` and a ``distance`` attribute.
+
+.. jupyter-execute::
+
+    telescope.observe(source=pulsar, distance=d_p)
+
+To simulate an observation of the pulsar scattered by the screen, we first
+use the :py:meth:`~screens.screen.Screen.observe` method from the screen to the
+pulsar, creating an object that encodes the images of the pulsar on the screen,
+and then generate an observation of the resulting object from the telescope.
+Note that the distance should be the relative distance from the object that is
+being observed to the object that does the observing.
+
+.. jupyter-execute::
+
+    obs_scr1_pulsar = scr1.observe(source=pulsar, distance=d_p-d_s)
+    obs1 = telescope.observe(source=obs_scr1_pulsar, distance=d_s)
+
+    print(obs1)
+
+Making an observation with :py:meth:`~screens.screen.Screen.observe` also gives
+access to a few key scintillometric quantities: the (complex) brightness of
+each path of radiation (the product of the magnifications of the source,
+screen, and telescope), the instantaneous geometric delay of the radiation
+following each path, and the time derivatives of those delays.
+
+.. jupyter-execute::
+
+    obs1.brightness
```

Comment:
Looking at the output of this, I guess `.brightness` should return a (dimensionless) `Quantity`!